source/_posts/2017-07-07-you-may-not-be-a-polyglot-but-your-code-needs-to-be.markdown

```diff
@@ -0,0 +1,120 @@
+
+I like to think of their relationship more like this: **i18N + L10N == G11N**
+While the W3 notes, "some people use other terms, such as globalization to refer to the same concept" as internationalization," I see the appropriate execution of internationalization and localization together as providing a globalized solution.
+
+Among examples of appropriately globalized solutions, TripAdvisor is one I like to highlight because it has a broad user base, and many of its users don�t realize it is used by others around the world in their own language. The following Trip Advisor elements help provide a positive user experience:
+*Flexing dates to show in the appropriate locale
+*Removing content that is not available in the target language (newsletters are a good example)
+*Accurate currency conversion
+*Allowing for text expansion to avoid truncation
+*Flexing logos and the brand to ensure that it is appropriate for the locale
+
+Another common misconception when creating globalized applications is that once something has been internationalized and translated, it can be shipped. The best way to highlight the folly of this perception is to look at some marketing mishaps. Below are some of my favorite ones:
+>A famous detergent from the Middle East, is white as snow, so it is called by the word for snow in Farsi. This doesn�t work well when it is referenced in English-speaking countries. In Farsi, the word for snow is barf.  
+>I also have a personal story. I come from a tropical country, and my native language is Spanish. When I first moved to the United States, I realized that I was not genetically engineered to be in the cold, dry weather of a Kansas City winter. My skin was itchy, and I desperately need some good hydrating lotion. My friends at school recommended something called Sarna. I was convinced they were trolling me because in Spanish, Sarna means Scabies.
+I have endured more than a dozen Kansas City winters, and I still can�t make myself use that lotion, because I can�t get over the name.
+
+Translation alone is not a solution. The translator must be proficient. In a previous position I was frequently asked how long it takes to learn a language. I was fighting the misconception that anyone who has taken two years of a language could translate. After failing to articulate why that was a misconception, I found a wonderful tool, thanks to the Foreign Language Institute and the Interagency Language Roundtable (ILR), who created a scale that is a set of descriptions of abilities to communicate in a language. One important aspect of this scale is that it is accurate when the evaluations are done by native speakers. It was originally developed by the United States Foreign Service Institute (FSI), and is still widely known as the FSI scale. It consists of descriptions of five levels of language proficiency.
+*Level 1: Elementary
```

Comment:
Minor, I think I may need to try reformatting some of this, as I expect some of this will need whitespace between these blocks of text and the unordered lists for formatting.